### PR TITLE
Associations

### PR DESCRIPTION
**Pull request by Lazersmoke**
This is untested because it refuses to read my `exceptions.yml` without NPE :/

### CODE REVIEW

**Comment by Lazersmoke** [comment]
Subsumes #7 and #8.

File: src/main/java/com/civclassic/altmanager/AltStorage.java [hunk 1]
```diff
@@ -10,22 +10,29 @@
 
 import vg.civcraft.mc.civmodcore.dao.ManagedDatasource;
 
-public class AltStorage {
+public class AltStorage extends ManagedDatasource {
```
Comment (TealNerd) [inline src/main/java/com/civclassic/altmanager/AltStorage.java]: why?

**Comment by Lazersmoke** [comment]
@TealNerd resolved

**Comment by Lazersmoke** [comment]
@TealNerd merge and put on test pls

**Pull request merged**